.github/workflows/benchmark.yml

```diff
@@ -31,10 +31,10 @@ jobs:
             ${{ runner.os }}-17-16-yarn-
 
       - name: Install Dependencies using Yarn
```

Comment:
@n1ru4l you can replace this whole section (node, cache, pnpm, pnpm i) with the setup action from shared-config.

Reply:
also in the other pipelines